docker/bluerange-compose.sh

```diff
@@ -206,6 +206,7 @@ EOF
   echo "       Kibana: https://${HOST}:${PORT_KIBANA:-5601} (admin / admin)"
   echo "        Minio: https://${HOST}:${PORT_MINIO:-9000} (${MINIO_ACCESS_KEY} / ${MINIO_SECRET_KEY})"
   echo "ElasticSearch: https://${HOST}:${PORT_ELASTICSEARCH:-9200} (admin / admin)"
+  echo "      Grafana: https://${HOST}:${PORT_GRAFANA:-9090} (admin / ${SYSTEM_ADMIN_PASSWORD})"
```

Comment:
Consider adding prometheus here as well (we do so for ElasticSearch and Kibana) as well.
Notice, the existing list was sorted by default port numbers.
Also, as you got it right now, Grafana is http (without no s).